api.md

```diff
@@ -1,17 +1,13 @@
 
-Hopper API
-----------
+# Hopper API
 
-This API is kept simple by adding a single command which allows mods to add
-containers like chests and furnaces to the hopper check list.
+This API provides two functions for adding containers such as chests and furnaces that will interact with hoppers.
 
+## hopper:add_container
 
-Command Usage
--------------
-
-Make sure any mods using this function has 'hopper' in the depends.txt file.
-
-hopper:add_container({ {"where_from", "node_name", "inventory_name"} })
+```lua
+hopper:add_container({ {"where_from", "node_name", "inventory_name"}, })
```

Comment:
`inventory_name` (or inventory name) is not a Minetest term, thus might confuse modders.

What does exist:

 * inventory (`get_list(name)`, `set_list(name, stacks)`)
 * inventory list name (a string)
 * inventory list (an integer-indexed "array" of `ItemStack`s)

Suggestion: `"list_name"` or `"inv_list_name"`

Whereas this was already documented, I do not see it being used anywhere. Or am I wrong about that?

Reply:
For that matter, I would also like to describe the variable "where_from" in a different way.
This is rather a relative location of the container and needs to be renamed to container_relative_location.

Reply:
But these corrections should not be in the current PR.

Reply:
These names are only descriptive example values in the documentation. They can be changed freely as long neither is exposed as an API for use in other mods.
In my previous comment I assumed that `inventory_name` *is exposed* in `hopper.groups[group[N]]`. However, I do not see any documented function related to hopper groups, thus it's OK to keep the naming as-is.